sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -202,100 +385,127 @@ private[ui] abstract class ExecutionTable(
         </td>
       }}
       {if (showSucceededJobs) {
-        <td>
-          {jobLinks(JobExecutionStatus.SUCCEEDED)}
-        </td>
-      }}
+      <td>
+        {jobLinks(JobExecutionStatus.SUCCEEDED)}
+      </td>
+    }}
       {if (showFailedJobs) {
-        <td>
-          {jobLinks(JobExecutionStatus.FAILED)}
-        </td>
-      }}
+      <td>
+        {jobLinks(JobExecutionStatus.FAILED)}
+      </td>
+    }}
     </tr>
   }
 
-  private def descriptionCell(
-      request: HttpServletRequest,
-      execution: SQLExecutionUIData): Seq[Node] = {
+  private def descriptionCell(execution: SQLExecutionUIData): Seq[Node] = {
     val details = if (execution.details != null && execution.details.nonEmpty) {
-      <span onclick="clickDetail(this)" class="expand-details">
+      <span onclick="this.parentNode.querySelector('.stage-details').
+      classList.toggle('collapsed')"
+            class="expand-details">
         +details
       </span> ++
-      <div class="stage-details collapsed">
-        <pre>{execution.details}</pre>
-      </div>
+        <div class="stage-details collapsed">
```

Comment:
I believe this shouldn't be indented more

Reply:
Yes. I have used a different scalastyle and checkstyle xml in IJ it seems.